mozci/console/commands/push.py

```diff
@@ -345,8 +386,13 @@ class ClassifyEvalCommand(Command):
         {--rev= : Head revision of the push.}
         {--from-date= : Lower bound of the push range (as a date in yyyy-mm-dd format or a human expression like "1 days ago").}
         {--to-date= : Upper bound of the push range (as a date in yyyy-mm-dd format or a human expression like "1 days ago"), defaults to now.}
-        {--medium-confidence= : If recalculate parameter is set, medium confidence threshold used to classify the regressions.}
-        {--high-confidence= : If recalculate parameter is set, high confidence threshold used to classify the regressions.}
+        {--intermittent-confidence-threshold= : Medium confidence threshold used to classify the regressions.}
+        {--real-confidence-threshold= : High confidence threshold used to classify the regressions.}
+        {--use-possible-regressions= : Use possible regressions while classifying the regressions.}
+        {--unknown-from-regressions= : Unknown from regressions while classifying the regressions.}
+        {--consider-children-pushes-configs= : Consider children pushes configs while classifying the regressions.}
+        {--cross-config-counts= : Cross-config counts used to classify the regressions.}
+        {--consistent-failures-counts= : Consistent failures counts used to classify the regressions.}
```

Comment:
The `*-counts` options don't seem to work from the CLI because they are tuples. We can handle it in a follow-up if you want, it's not a big priority.